src/index.html

```diff
@@ -1,13 +1,440 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Bakery</title>
+    <link rel="icon" href="./images/Icon-cookie.svg">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
-  </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
     <script type="text/javascript" src="scripts/main.js"></script>
+  </head>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <nav class="header__nav nav">
+          <a href="#" class="logo">
+            <img
+              src="./images/Logo.svg"
+              alt="Bakery"
+              class="logo__image"
+            >
+          </a>
+          <div class="container--hidden">
+            <ul class="nav__menu">
+              <li class="nav__item">
+                <a href="#products" class="nav__link">Products</a>
+              </li>
+              <li class="nav__item">
+                <a href="#aboutUs" class="nav__link">About Us</a>
+              </li>
+              <li class="nav__item">
+                <a href="#Blog" class="nav__link">Blog</a>
+              </li>
+              <li class="nav__item nav__item--last">
+                <a href="#contactUs" class="nav__link nav__link--last">Contact us</a>
+              </li>
+            </ul>
+          </div>
+          <a href="#menu" class="header__menu-opener"></a>
+        </nav>
+
+
+        <div class="grid grid--desktop">
+          <p class="
+                  header__about
+                  grid__item
+                  grid__item--onTablet--1-4
+                  grid__item--onDesKtop--1-2"
+          >
```

Comment:
Better add the same changes all over the document.
```suggestion
          <p class="
             header__about
             grid__item
             grid__item--onTablet--1-4
             grid__item--onDesKtop--1-2"
          >
```